platform.js

```diff
@@ -403,10 +403,11 @@ module.exports = {
 
     return module.exports.spawn(command, args);
   },
-  getFreeDiskSpace() {
+  getFreeDiskSpace(dir) {
     return new Promise((resolve, reject)=>{
+      dir = (dir) ? dir : module.exports.getInstallDir();
```

Comment:
@rodrigopavezi You should be able to do `getFreeDiskSpace(dir=module.exports.getInstallDir()) {` and get rid of line 408